PartyWebApp/Controllers/PerformersController.cs

```diff
@@ -54,7 +60,7 @@ public IActionResult Create()
         // more details, see http://go.microsoft.com/fwlink/?LinkId=317598.
         [HttpPost]
         [ValidateAntiForgeryToken]
-        public async Task<IActionResult> Create([Bind("Id,firstName,lastName,genre,rating,imageUrl")] Performer performer)
+        public async Task<IActionResult> Create([Bind("Id,SpotifyId")] Performer performer)
```

Comment:
Why did you delete the this parameters?

Reply:
Because I've changed the model and you don't need those parameters anymore, just the SpotifyId.